### PR TITLE
Email is the first step

### PR DESCRIPTION
**Pull request by dmurko**

### CODE REVIEW

**Comment by cas--** [comment]
I don't agree, this is a step that requires more privileges than the other steps so would be a blocker for example marbe who is in peopleops.

**Comment by dmurko** [comment]
@zupo It's in the onboarding checklist: https://github.com/niteoweb/operations/pull/367 (new PR)

**Comment by cas--** [comment]
I have made further improvements to the emails so just another quick review required.

File: onboarding.md [hunk 1]
```diff
@@ -30,26 +30,22 @@ That is it! Go to your onboarding issue and start checking things off!
...
 %0D%0A
-%0D%0A
-Regards%0D%0A
-%0D%0A
-" target="_blank">
-Welcome to Niteo
-</a>email.
+Regards%0D%0A%0D%0A
+">Welcome to Niteo email</a> and the request for their GitHub handle.
+* Add the new team member to the [Trialists] group, allowing access to [operations](https://github.com/niteoweb/operations) and [handbook](https://github.com/niteoweb/handbook) repositories.
```
Comment (zupo) [inline onboarding.md]: This needs to be the first task, otherwise when you send the email, the trialist won't be able to access their onboarding ticket.

Comment (dmurko) [reply onboarding.md]: @zupo The content of the email changed. Me and @cas-- did the changes based on my issues with the process when I onboarded the two new designers.

Comment (zupo) [reply onboarding.md]: Got it, sry for the noise.

**Pull request merged**